components/gitpod-db/src/user-db.ts

```diff
@@ -32,13 +32,6 @@ export interface UserDB extends OAuthUserRepository, OAuthTokenRepository {
     findUserById(id: string): Promise<MaybeUser>;
     findUserByIdentity(identity: IdentityLookup): Promise<MaybeUser>;
 
-    /**
-     * Gets the number of users.
-     *
-     * @param excludeBuiltinUsers substract the builtin-users from the count (currently only the user builtin-workspace-prober), true by default
-     */
-    getUserCount(excludeBuiltinUsers?: boolean): Promise<number>;
```

Comment:
This is being used in https://github.com/gitpod-io/gitpod/pull/17718/files#diff-315103e87ae1b7ee06fb5f3f99090d11a3748704f3799561ef3dd49d73ea0e33R18 AFAIK

Reply:
Thanks for pointing that out. 
I'll revert.